observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -543,7 +626,7 @@ object TargetObsList {
                             clazz = ExploreStyles.DeleteButton |+| ExploreStyles.JustifyRight,
                             onClickE = (e: ReactMouseEvent, _: ButtonProps) =>
                               e.stopPropagationCB *>
-                                deleteTarget(targetId, undoCtx.setter, focusOnDelete)
+                                deleteTarget(targetId, undoCtx.setter, focusOnDelete).runAsyncCB
```

Comment:
The above is the last remaining `*>` if you want to change  it to `>>` for consistency.

Reply:
:+1: thank you